cmp/chart/impl/ChartContextMenuItems.ts

```diff
@@ -0,0 +1,201 @@
+            };
+        }
+    }
+
+    return item as MenuItem;
+}
+
+function parseToken(token: string, chartModel: ChartModel): MenuItem | string {
+    switch (token) {
+        case 'viewFullscreen':
+            return {
+                text: 'View in full screen',
+                icon: Icon.expand(),
+                hidden: !chartModel,
+                actionFn: () => chartModel.highchart.fullscreen.toggle()
+            };
+        case 'copyToClipboard':
+            return {
+                text: 'Copy to clipboard',
+                icon: Icon.copy(),
+                hidden: !chartModel || !Highcharts.isWebKit,
+                actionFn: () => chartModel.highchart.copyToClipboardAsync()
+            };
+        case 'printChart':
+            return {
+                text: 'Print chart',
+                icon: Icon.print(),
+                hidden: !chartModel,
+                actionFn: () => chartModel.highchart.print()
+            };
+        case 'downloadJPEG':
+            return {
+                text: 'Download JPEG image',
+                icon: Icon.fileImage(),
+                hidden: !chartModel,
+                actionFn: () =>
+                    chartModel.highchart.exportChartLocal({
+                        type: 'image/jpeg'
+                    })
+            };
+        case 'downloadPNG':
+            return {
+                text: 'Download PNG image',
+                icon: Icon.fileImage(),
+                hidden: !chartModel,
+                actionFn: () => chartModel.highchart.exportChartLocal()
+            };
+        case 'downloadSVG':
+            return {
+                text: 'Download SVG vector image',
+                icon: Icon.fileImage(),
+                hidden: !chartModel,
+                actionFn: () =>
+                    chartModel.highchart.exportChartLocal({
+                        type: 'image/svg+xml'
+                    })
+            };
+        case 'downloadPDF':
+            return {
+                text: 'Download PDF',
+                icon: Icon.fileImage(),
+                hidden: !chartModel,
+                actionFn: () =>
+                    chartModel.highchart.exportChartLocal({
+                        type: 'application/pdf'
+                    })
+            };
+        case 'downloadCSV':
+            return {
+                text: 'Download CSV',
+                icon: Icon.fileCsv(),
+                hidden: !chartModel,
+                actionFn: () => chartModel.highchart.downloadCSV()
+            };
+        case 'downloadXLS':
+            return {
+                text: 'Download Excel',
+                icon: Icon.fileExcel(),
+                hidden: !chartModel,
+                actionFn: () => chartModel.highchart.downloadXLS()
+            };
+        default:
+            return token;
+    }
+}
+
+function isMenuItem(item: ChartMenuItemLike): item is MenuItem {
+    return !isString(item) && !isValidElement(item);
```

Comment:
This is called in just one place here, and we've already checked if the value is a string and won't reach here if so - could inline the validElement check and drop this function which I think just adds a bit more noise to the file at the moment